### DISCUSSION TITLE
Converting Forge server to Fabric, lost Overworld-Nether portals, how to keep portal data?

### DISCUSSION THREAD

**Discussion by Zxurian** [iPortalTeam/ImmersivePortalsMod#730] [Q&A]
so I have an existing Forge server that I want to convert to a fabric server. In the process of switching over, Portals created between overworld locations remained intact, but portals created between overworld and nether were lost.

Trying to relight the obsidian frames doesn't re-connect them to where they were previously, but instead creates an entirely new portal/frame in the Nether.

How can I maintain the portals / portal data of overworld-nether portals (or overworld-end for that matter) within the new fabric instance of the server?

**Comment by qouteall** [comment]
Forge and fabric have different block id registry. So the forge world's portal placeholder blocks are not recognized in fabric. Currently there is no easy solution,

**Comment by Zxurian** [comment]
thank you for the advice. We've decided to go the "write down coordinates
of all our portals and manually recreate them" method.
